src/main/java/com/joseeduardo/todoist/entity/enums/Status.java

```diff
@@ -1,5 +1,23 @@
 package com.joseeduardo.todoist.entity.enums;
 
 public enum Status {
-	CRIADA, INICIADA, CONCLUIDA;
+
+	CRIADA("CRIADA"), INICIADA("INICIADA"), CONCLUIDA("CONCLUIDA");
```

Comment:
Se eu não me engano, não precisa replicar o mesmo nome para o valor de cada tipo enumerado. Pode manter como estava antes que ele funciona, se não me engano.

Reply:
CRIADA, INICIADA, CONCLUIDA;

Reply:
Só pular uma linha entre elas:
```suggestion
	CRIADA, 
	INICIADA, 
	CONCLUIDA;
```